bmeg_app/views/literature_support_view.py

```diff
@@ -48,90 +44,134 @@
     html.Hr(),
     dbc.Row(
         [
-            dbc.Col(dcc.Loading(id='occr', type="default",children=html.Div()),width=3,style={"c":"100%"}),
-            dbc.Col(dcc.Loading(id='resp_histo', type="default",children=html.Div()),width=4,style={"height":"100%"}),
-            dbc.Col(dcc.Loading(id='pie_taxon', type="default",children=html.Div()),width=5,style={"height":"100%"}),
-  
+            dbc.Col(dcc.Loading(id='occr', children=html.Div()),width=3,style={"c":"100%"}),
+            dbc.Col(dcc.Loading(id='resp_histo', children=html.Div()),width=4,style={"height":"100%"}),
+            dbc.Col(dcc.Loading(id='pie_taxon', children=html.Div()),width=5,style={"height":"100%"}),
+
         ],
     ),
     html.Hr(),
-    dcc.Loading(id='evd', type="default",children=html.Div()),
+    dcc.Loading(id='evd', children=html.Div()),
 ],style={'fontFamily': styles['t']['type_font']})
 
 
 @app.callback(
-    Output("evd", "children"),
+    dash.dependencies.Output('gene_dd', 'options'),
+    [dash.dependencies.Input('gene_dd', 'search_value')]
+)
+def update_options(search_value):
+    """Lookup the search value in elastic."""
+    if not search_value:
+        raise PreventUpdate
+    genes = gene_search(search_value)
+    return genes
+
+@app.callback(
+    [Output("evd", "children"), Output("resp_histo", "children"),Output("pie_taxon", "children"), Output("occr", "children")],
     [Input('gene_dd', 'value')]
 )
-def render_callback(selected_gene):
+def build_evidence_table(search_value):
+    gene = search_value.split("/")[1]
     '''Occurance table'''
-    # Filter for gene
-    df = base_df[base_df['geneID']==selected_gene].reset_index(drop=True)
-    # Generate table
-    dictionary = lsu.parse_src_doc(df, 'litETC',['cancerType','drugAbstracts'])
-    df = lsu.build_publication_table(dictionary)
-    dashtable = dash_table.DataTable(
-        id='export_pub_table',
-        data = df.to_dict('records'),
-        columns=[{"name": i, "id": i} for i in df.columns],
+    ## associations
+    mapping = {
```

Comment:
this is a better way to run queries and prep for df storage. we should update queries for other widgets with this format